src/machinetalk/videoserver/videoserver.py

```diff
@@ -180,6 +182,8 @@ def main():
     mki = configparser.ConfigParser()
     mki.read(mkini)
     mk_uuid = mki.get("MACHINEKIT", "MKUUID")
+    aformat = mki.has_option("MACHINEKIT", "ANNOUNCE_FORMAT") and \
+        mki.get("MACHINEKIT", "ANNOUNCE_FORMAT") or None
```

Comment:
It's probably something python specific, but why every `.py` service implements it's own `machinekit.ini` pattern lookup, i.e. the same code is implemented in multiple source files? Wouldn't it be better implemented in `service.py`?

Reply:
It seems to be a relic, and I was avoiding totally refactoring them. The c interface does move the ini parsing to the service object, and I like it better. I’ll take a look and see if it’s practical or if we should make that refactoring it’s own issue.

Reply:
OK. It's just  "clean code" issue. It has no functional meaning.

Reply:
Commit coming to address this duplication. Each service will basically call into the ini parsing code similar to the c-style service mechanism, and hopefully make the code a little more DRY friendly.

Reply:
Great. Sometime in the future it would probably be best and go completely away with this python code. Just create python wrapper called Service around C code.

Reply:
I've moved the ini parsing to the service.py file just for common editing. I wasn't aware of the other options for implementing the machinetalk service stuff, and when I tried to do more the changes quickly became viral. I think the refactor should definitely be done in it's own issue.

I had to move those changes to another branch and rebase on @machinekoder's latest updates. I will run the Black formatter as suggested and make a new pull request after addressing the latest concerns.